src/kirby/components/page/page.component.ts

```diff
@@ -163,6 +168,7 @@ export class PageComponent implements OnInit, OnDestroy, AfterContentInit, After
 
   ngOnInit(): void {
     this.url = this.router.url;
+    this.enter.emit();
```

Comment:
This doesn't make it fire twice?

Reply:
No because, the ngOnInit only run once, and after that the emit in router events subscriptions take over.